zerver/lib/bugdown/__init__.py

```diff
@@ -302,6 +303,15 @@ def get_tweet_id(url):
         return None
     return tweet_id_match.group("tweetid")
 
+def render_tex(tex):
+    # type: (Text) -> Text
+    katex = Popen(['npm', 'run', '-s', 'katex'], stdin=PIPE, stdout=PIPE)
+    stdout = katex.communicate(input=tex)[0]
+    if katex.returncode == 0:
+        return stdout
+    else:
+        return None
+
```

Comment:
`subprocess.check_output("cat", input=tex)` (well, need to make sure it's bytes, not text)

But it appears this isn't available in Python 2.  I guess we can leave it as-is for now :)